pymc3_models/models/NaiveBayes.py

```diff
@@ -0,0 +1,269 @@
+import functools as ft
+import numpy as np
+import scipy.stats
+import theano
+from sklearn.metrics import accuracy_score
+import pymc3 as pm
+
+from pymc3_models.exc import PyMC3ModelsError
+from pymc3_models.models import BayesianModel
+
+
+__all__ = ["GaussianNaiveBayes"]
+
+
+
+# Gaussian Naive Bayes
+# -----------------------------------------------------------------------------
+
+class GaussianNaiveBayes(BayesianModel):
+    """ Naive Bayes classification built using PyMC3.
+
+    The Gaussian Naive Bayes algorithm assumes that the random variables
+    describing each class and each feature are independent and distributed
+    according to independent Normal distributions.
+
+    See the documentation of the `create_model` method for details on the model
+    itself.  
+    
+    """
+
+    def __init__(self):
+        super(GaussianNaiveBayes, self).__init__()
+
+    def create_model(self):
+        """ Creates and returns the PyMC3 model.
+
+        Model
+        -----
+
+        We note $x_{jc}$ the value of the j-th element of the data vector $x$
```

Comment:
Were you able to get the latex to render locally? My cursory research suggests we may need to put some prefixes to get Sphinx to pick up on it (http://www.sphinx-doc.org/en/stable/ext/math.html).

Reply:
You were right, I changed the prefix and Latex equations now render ok. The only issue I have with it is that it's very ugly; I'll try to see if we can change that by tweaking the settings.